fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/logical/PushdownLimit.java

```diff
@@ -40,24 +39,32 @@
  * <p>
  * Limit can't be push down if it has a valid offset info.
  */
-public class LimitPushDown implements RewriteRuleFactory {
+public class PushdownLimit implements RewriteRuleFactory {
 
     @Override
     public List<Rule> buildRules() {
         return ImmutableList.of(
                 // limit -> join
                 logicalLimit(logicalJoin(any(), any())).whenNot(Limit::hasValidOffset)
```

Comment:
maybe do not need `join(any(), any())`? `join()` is enough

Reply:
```java
f (newJoin == null || limit.child().children().equals(newJoin.children())) {
                                return null;
                            }
```

use it